packages/machine-extractor/examples/transition-description.machine.ts

```diff
@@ -0,0 +1,23 @@
+import { createMachine } from "xstate";
+
+export const transitionDescription = createMachine({
+	id: "TransitionDescriptionMachine",
+	type: "parallel",
+	states: {
+		Ticket: {
+			initial: "Open",
+			states: {
+				Open: {
+					on: {
+						cancel: {
+							actions: "cancel ticket",
+							target: "Canceled",
+							description: "This is description example",
+						},
+					},
+				},
+				Canceled: {},
+			},
+		},
+	},
+});
```

Comment:
Converted to use spaces for formatting:

```suggestion
export const transitionDescription = createMachine({
  id: "TransitionDescriptionMachine",
  type: "parallel",
  states: {
    Ticket: {
      initial: "Open",
      states: {
        Open: {
          on: {
            cancel: {
              actions: "cancel ticket",
              target: "Canceled",
              description: "This is description example",
            },
          },
        },
        Canceled: {},
      },
    },
  },
});
```